test_regress/t/t_sampled_expr_unsup.out

```diff
@@ -0,0 +1,11 @@
+%Error-UNSUPPORTED: t/t_sampled_expr_unsup.v:36:36: Unsupported: Assignment to variable inside sampled expression
+                                                  : ... In instance t.t1
+   36 |    assert property (@(posedge clk) a++ >= 0);
+      |                                    ^
+                    ... For error description see https://verilator.org/warn/UNSUPPORTED?v=latest
+%Warning-UNSIGNED: t/t_sampled_expr_unsup.v:36:40: Comparison is constant due to unsigned arithmetic
```

Comment:
Please get rid of this warning. Preferably by using a different type/comparison, but I'll accept `-Wno-UNSIGNED`.

Reply:
I opt for `-Wno-UNSIGNED` because I intended the test to be tautological for it not to fail for any other reason than just "write in sampled".

I updated the test.